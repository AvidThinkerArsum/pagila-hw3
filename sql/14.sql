/*
 * Management also wants to create a "best sellers" list for each category.
 *
 * Write a SQL query that:
 * For each category, reports the five films that have been rented the most for each category.
 *
 * Note that in the last query, we were ranking films by the total amount of payments made,
 * but in this query, you are ranking by the total number of times the movie has been rented (and ignoring the price).
 */
