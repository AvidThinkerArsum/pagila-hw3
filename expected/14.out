    name     |         title          | total rentals 
-------------+------------------------+---------------
 Action      | GOODFELLAS SALUTE      |            31
 Action      | TIMBERLAND SKY         |            31
 Action      | CAT CONEHEADS          |            30
 Action      | HARRY IDAHO            |            30
 Action      | IDOLS SNATCHERS        |            30
 Animation   | GRIT CLOCKWORK         |            32
 Animation   | ENGLISH BULWORTH       |            30
 Animation   | SUSPECTS QUILLS        |            30
 Animation   | WITCHES PANIC          |            30
 Animation   | CONFIDENTIAL INTERVIEW |            29
 Children    | JUGGLER HARDLY         |            32
 Children    | DOGMA FAMILY           |            30
 Children    | FAMILY SWEET           |            29
 Children    | GREATEST NORTH         |            29
 Children    | TITANS JERK            |            29
 Classics    | HOBBIT ALIEN           |            31
 Classics    | TIMBERLAND SKY         |            31
 Classics    | HARRY IDAHO            |            30
 Classics    | MARRIED GO             |            30
 Classics    | MASSACRE USUAL         |            30
 Comedy      | FORWARD TEMPLE         |            32
 Comedy      | GRIT CLOCKWORK         |            32
 Comedy      | JUGGLER HARDLY         |            32
 Comedy      | ROBBERS JOON           |            31
 Comedy      | ENGLISH BULWORTH       |            30
 Documentary | RIDGEMONT SUBMARINE    |            32
 Documentary | APACHE DIVINE          |            31
 Documentary | GOODFELLAS SALUTE      |            31
 Documentary | NETWORK PEAK           |            31
 Documentary | RUSH GOODFELLAS        |            31
 Drama       | BUCKET BROTHERHOOD     |            34
 Drama       | FORWARD TEMPLE         |            32
 Drama       | HOBBIT ALIEN           |            31
 Drama       | IDOLS SNATCHERS        |            30
 Drama       | MASSACRE USUAL         |            30
 Family      | ROCKETEER MOTHER       |            33
 Family      | SCALAWAG DUCK          |            32
 Family      | APACHE DIVINE          |            31
 Family      | BUTTERFLY CHOCOLAT     |            30
 Family      | GRAFFITI LOVE          |            30
 Foreign     | GOODFELLAS SALUTE      |            31
 Foreign     | NETWORK PEAK           |            31
 Foreign     | WIFE TURN              |            31
 Foreign     | STORM HAPPINESS        |            29
 Foreign     | SWEETHEARTS SUSPECTS   |            29
 Games       | BUCKET BROTHERHOOD     |            34
 Games       | ROBBERS JOON           |            31
 Games       | DOGMA FAMILY           |            30
 Games       | FROST HEAD             |            30
 Games       | PULP BEVERLY           |            30
 Horror      | RIDGEMONT SUBMARINE    |            32
 Horror      | CAT CONEHEADS          |            30
 Horror      | PULP BEVERLY           |            30
 Horror      | BOOGIE AMELIE          |            29
 Horror      | GLEAMING JAWBREAKER    |            29
 Music       | BUCKET BROTHERHOOD     |            34
 Music       | SCALAWAG DUCK          |            32
 Music       | FROST HEAD             |            30
 Music       | MUSCLE BRIGHT          |            30
 Music       | RUGRATS SHAKESPEARE    |            30
 New         | FORWARD TEMPLE         |            32
 New         | APACHE DIVINE          |            31
 New         | FROST HEAD             |            30
 New         | GRAFFITI LOVE          |            30
 New         | SHOCK CABIN            |            30
 Sci-Fi      | ZORRO ARK              |            31
 Sci-Fi      | HARRY IDAHO            |            30
 Sci-Fi      | IDOLS SNATCHERS        |            30
 Sci-Fi      | MASSACRE USUAL         |            30
 Sci-Fi      | SUSPECTS QUILLS        |            30
 Sports      | HOBBIT ALIEN           |            31
 Sports      | GRAFFITI LOVE          |            30
 Sports      | MUSCLE BRIGHT          |            30
 Sports      | CONFIDENTIAL INTERVIEW |            29
 Sports      | TRIP NEWTON            |            28
 Travel      | WIFE TURN              |            31
 Travel      | DOGMA FAMILY           |            30
 Travel      | MARRIED GO             |            30
 Travel      | TALENTED HOMICIDE      |            29
 Travel      | TITANS JERK            |            29
(80 rows)

